projects/aws/eks-a-admin-image/eks-a-admin.pkr.hcl

```
@@ -0,0 +1,238 @@
+packer {
+  required_version = ">= 1.8.0"
+  required_plugins {
+    amazon = {
+      version = ">= 1.0.0"
+      source  = "github.com/hashicorp/amazon"
+    }
+    vsphere = {
+      version = ">= v1.0.3"
+      source  = "github.com/hashicorp/vsphere"
+    }
+  }
+}
+
+variable "build_username" {
```

nit: could we follow the same naming standard as the other vars?
```suggestion
variable "build-username" {
```